global/scripts/mppmodecheck.shlib

```diff
@@ -0,0 +1,56 @@
+# Functions for checking compliance with selected MPP mode for HCP Pipelines
```

Comment:
Similar to the debugging shlib, I'd suggest the first line be `#!/bin/echo This script should not run directly:`

Reply:
added.